shell/components/nav/Group.vue

```diff
@@ -293,7 +293,8 @@ export default {
 
     > A {
       display: block;
-      padding-left: 16px;
+      box-sizing:border-box;
+      height: 33px;
```

Comment:
Would we be able to specify a height of `100%` since our header already has a height of `33px`?

Reply:
That works too, you are right that it's better in case outer height changes